### PR TITLE
avm2: Implement Matrix3D.copyColumn*() as native methods

### PR DESCRIPTION
**Pull request by kjarosh**
## Description

This implements Matrix3D.copyColumn*() methods as native and handles edge cases properly.

## Testing

Tests included.

## Checklist

- [x] I, a human, have self-reviewed this PR and fully understand the changes within.
- [x] I have [made or updated tests](https://github.com/ruffle-rs/ruffle/blob/master/CONTRIBUTING.md#test-guidelines) where possible.
- [x] All of my commits are properly scoped, compile successfully, and pass all tests.
- [x] This PR does not make sense to split up into smaller PRs.
- [x] An LLM was involved in the authoring of this code.

### CODE REVIEW

File: core/src/avm2/globals/flash/geom/matrix_3d.rs [hunk 1]
```diff
@@ -128,6 +128,64 @@ pub fn prepend<'gc>(
...
+) -> Result<Value<'gc>, Error<'gc>> {
+    let this = this.as_object().unwrap();
+    let column = args.get_u32(0);
+    let vector3d = args.get_object(activation, 1, "vector3D")?;
+
+    if column > 3 {
+        return Err(make_error_2004(activation, Error2004Type::ArgumentError));
+    }
+
+    let [x, y, z, w] = [
```
Comment (Lord-McSweeney) [inline core/src/avm2/globals/flash/geom/matrix_3d.rs]: This should use `read_vector3d`, right?

Comment (kjarosh) [reply core/src/avm2/globals/flash/geom/matrix_3d.rs]: It originally did, but I would have to wrap it in `Option` and `Value` and then unwrap the result, so I chose to use slots directly, similarly to setters.

Comment (Lord-McSweeney) [reply core/src/avm2/globals/flash/geom/matrix_3d.rs]: IMO it would be better then to make `read_vector3d` take an `Object` directly and make its other callers responsible for unwrapping `Option<Value>`.

Comment (kjarosh) [reply core/src/avm2/globals/flash/geom/matrix_3d.rs]: Added a separate function

**Review by Lord-McSweeney — APPROVED** [review]
